src/popup/components/ToggleMultisigButton.vue

```diff
@@ -65,7 +65,7 @@ export default defineComponent({
       if (!multisigAccounts.value?.length && props.isMultisig) {
         toggleMultisigDashboard(false);
       }
-    });
+    }, { deep: true });
```

Comment:
The goal of the wathcer here is to toggle multisigDashboard back to general accounts if there are no multisig ones. So there is no need in the deep option here.

Reply:
This is related to breaking change in Vue 3: https://v3-migration.vuejs.org/breaking-changes/watch.html
So in my opinion the change is correct, but I can't be 100% sure.

Reply:
Also I consider having this watcher and logic for switching from multisig here is a bad thing. That's a tiny component deep in the structure of the APP and it's responsible for sucha big thing. I remember first time I was searching for the logic that was redirecting me from the multisig and I didn't found it. @pstachel eventually helped me as he was the author of this. Maybe we should think about moving this somewhere else? @pstachel @CedrikNikita

Reply:
@CedrikNikita you are correct, the deep option is not needed here. I've updated the PR